extension-base/src/main/java/com/azure/autorest/extension/base/jsonrpc/Connection.java

```diff
@@ -52,7 +52,7 @@ public Connection(OutputStream writer, InputStream input) {
         this.requestId = new AtomicInteger(0);
     }
 
-    private boolean isAlive = true;
+    private volatile boolean isAlive = true;
```

Comment:
Not related, just to prevent JIT from over optimization:
https://github.com/Azure/autorest.java/blob/6fabb93051072929ea2641bcae63a3276fa7fd50/extension-base/src/main/java/com/azure/autorest/extension/base/jsonrpc/Connection.java#L157